src/index.css

```diff
@@ -1,41 +1,26 @@
 /*This css file is only used for classes used very often and across multiple pages, as well as some global styles.*/
 
-/*font imports*/
 @font-face {
-    /*Inter: basic font for most text*/
-    font-family: 'Inter';
-    font-weight: 300;
-    src: url('./fonts/inter/Inter-VariableFont_slnt\,wght.ttf');
+    /* Open Sans: default font */
```

Comment:
I prefer the old font a lot over the new one, especially since it makes the UI stand out less, since the font is less bold and it is harder to look at the UI since a lot more white space than before

Reply:
Ich denke dass sich die Schriftart aber vielleicht eignet bei Dingen die nicht content sind, also vielleicht wäre eine Option alles was der user erstellt ist in der alten font und alles was vom System ist, ist in der neuen

Reply:
Habe die Schriftart an einigen Stellen geändert.